### ISSUE TITLE
Psi spellcaster interfaces and Ars Magica 2 Inscription Tables sometimes lose data

### ISSUE THREAD

**Issue by Sunconure11** [SpongePowered/SpongeForge#1500]
A similar issue happens with Ars Magica 2, in particular, with the inscription table, preventing players from saving their spells to be worked on later.

This issue seems to be common on my spongeforge-based server, as reported by various players. However, I cannot get enough details, unfortunately, though I've seen it with Psi, spells will sometimes refuse to apply themselves to bullets and what not, and the spellcaster interface will sometimes blank itself.

Forge Version: 1.10.2 - 12.18.3.2281
Sponge Version: 1.10.2-2281-5.2.0-BETA-2317

**Comment by gabizou** [comment]
Half of this has no meaning to us Sponge developers. IF you can explain what you mean by "the spell caster interface will sometimes blank itself." and "spells will refuse to apply themselves to bullets", because this has little meaning otherwise.

**Comment by Eufranio** [comment]
The spellcaster interface is a block added by the Psi mod, you can build spells there (adding effects and stuff). He wants to say that this block loses its data sometimes, but I personally dont have this issue in any of my servers.

**Comment by Sunconure11** [comment]
@Eufranio is correct. Something similar happens to the Inscription Table from Ars Magica 2 (according to some players) as well. I have no clue WTF is going on.

I'm sorry for the vague nature, as I cannot get any more info aside from literally that. I managed to replicate the issue with Psi, but it's still kind of vague in terms of what info I can get from it. No crashes. No abnormal logs.

I feel that even giving a log would be useless, as nothing would show, but if you feel it may help, I can get one from around the time such an issue occurs.

Modlister info, just in case something stands out, in terms of known bugs: http://www.modlister.com/user/Sunconure11

Edit 1: Finally got some info regarding the inscription table. I think that might just be a case of user error, I'll have someone else look at that to confirm it. But I did manage to replace the Psi issue on the server. I'll look into both myself once more tomorrow, due to the auth servers being down.

Edit 2: The issue is GUI-only, for getting the spell onto a bullet/equivalent for Psi works after you create a valid spell, even if it disappears from the GUI. However, it seems loading a spell takes longer than normal, as a result of this, requiring one to shift-click a few times while holding the bullet/whatever, at least, from my investigation into the bug. AM2 bug will be looked into next. Like I said, this issue is rather vague in nature, despite me repeating it. I could just be an idiot replicating another idiot's work for all I know. This issue is rather hard to convey, I apologize.
